.github/workflows/build-doc-app.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
           cp -r ./docs/gh-pages-proj/dist/* ./docs/
 
       - name: Add & Commit
-        uses: EndBug/add-and-commit@v7.2.1
+        uses: EndBug/add-and-commit@v8.0.1
```

Comment:
If you always want the latest version just change this to `latest`.

Reply:
pinning is much better as you are explicit about what you are pulling down / you know when you are potentially pulling in a breaking change (like this 7.X.X -> 8.X.X)